src/main/java/test/project/bookshop/dto/book/BookRequestDto.java

```diff
@@ -19,7 +20,8 @@ public class BookRequestDto {
     @NotBlank
     private String description;
     private String coverImage;
-    @Min(0)
+    @Positive
```

Comment:
Check that this annotation works well with BigDecimal

Reply:
I don't find any issues